### PR TITLE
[plugin.video.gamegurumania] 1.0.7+matrix.1

### PR DESCRIPTION
**Pull request by skipmodea1**
### Description
Made a python 3 version based upon my python 2 version.
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.gamegurumania/addon.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">Watch game videos made from ggmania.com</summary>
+	<description lang="en_GB">Watch game videos made from ggmania.com</description>
+	<disclaimer lang="en_GB">For bugs, requests or general questions visit the ggmania.com thread on the Kodi forum.</disclaimer>
+	<summary lang="nl_NL">Bekijk game videos van ggmania.com</summary>
+	<description lang="nl_NL">Bekijk game videos van ggmania.com</description>
+	<disclaimer lang="nl_NL">Bugs of andere feedback op deze plugin kan geplaatst worden in de ggmania.com thread op het Kodi forum.</disclaimer>
+    <language>en</language>
+    <platform>all</platform>
+    <license>GNU GENERAL PUBLIC LICENSE. Version 3, June 2007</license>
```
Comment (enen92) [inline plugin.video.gamegurumania/addon.xml]: Spdx identifier if possible

File: plugin.video.gamegurumania/resources/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+# Enables relative imports
```
Comment (enen92) [inline plugin.video.gamegurumania/resources/__init__.py]: In python3 your init.py placeholder files shouldn't be needed

**Review by enen92 — CHANGES_REQUESTED** [review]
Only the spdx is needed. The init stuff isn't needed as probably you have a single codebase for the 2 versions

File: plugin.video.gamegurumania/addon.xml (hunk 1 above)
Comment (skipmodea1) [reply plugin.video.gamegurumania/addon.xml]: Fixed

File: plugin.video.gamegurumania/resources/__init__.py (hunk 2 above)
Comment (skipmodea1) [reply plugin.video.gamegurumania/resources/__init__.py]: Fixed

**Review by enen92 — APPROVED** [review]

**Pull request merged**